CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 Unreleased changes are available as `avenga/couper:edge` container.
 
+* **Added**
+  * Unbeta [OIDC](./docs/REFERENCE.md#oidc-block) access control ([#400](https://github.com/avenga/couper/pull/400))
```

Comment:
As the `oidc` block does not only configure an access control but also the `beta_oauth_authorization_url()` function, I would prefer using "block" instead of "access control".

Reply:
Done